### PR TITLE
community/events: Add troubleshoot rook-ceph talk

### PR DESCRIPTION
**Pull request by Thingee**
Signed-off-by: Mike Perez <thingee@gmail.com>

### CODE REVIEW

File: src/en/community/events/2022/ceph-virtual/index.md [hunk 1]
```diff
@@ -277,9 +277,21 @@ Cybozu has been running and testing their Rook/Ceph clusters for two years. Duri
    </td>
    <td><strong>DATA SECURITY AND STORAGE HARDENING IN ROOK AND CEPH</strong>
 <p>
-1. The NIC SR-IOV is used. Each OSD uses an exclusive VF NIC. 2. The DPDK interrupt mode is added. 3. The single-CPU core and multiple NIC queues are implemented to improve performance. 4. The admin socket command is added to obtain the NIC status, collect statistics, and locate faults. 5. Adjust the CEPH throttling parameters, TCP, and DPDK packet sending and receiving buffer sizes to prevent packet loss and retransmission. 6. The Crimson message component uses the Seastar DPDK.
+This session shares an overview of troubleshooting a rook-ceph cluster, discussing some troubleshooting scenarios. This is followed by an introduction and an overview of kubectl-rook-ceph krew plugin, and how it makes managing containers easier from a troubleshooting perspective.
+We’ll also discuss future issues we’re planning to solve with it here, followed by a short future roadmap of the rook project.
+Later, we look forward to discussing and gathering feedback from the users about common and challenging problems they face while troubleshooting the clusters.
    </td>
    <td>Federico Lucifredi
```
Comment (jdurgin) [inline src/en/community/events/2022/ceph-virtual/index.md:284]: Is the data security and storage hardening talk still meant to be here? It doesn't look like the description/author match for that one

Comment (jdurgin) [reply src/en/community/events/2022/ceph-virtual/index.md:284]: looks like the description is accidentally from today's "Optimize Ceph messenger Performance" talk

Comment (Thingee) [reply src/en/community/events/2022/ceph-virtual/index.md:282]: Whoops, thanks!

File: src/en/community/events/2022/ceph-virtual/index.md [hunk 2]
```diff
@@ -275,11 +275,21 @@ Cybozu has been running and testing their Rook/Ceph clusters for two years. Duri
...
+We explore the security model exposed by Rook with Ceph, the leading software-defined storage platform of the Open Source world. Digging increasingly deeper in the stack, we examine hardening options for Ceph storage appropriate for a variety of threat profiles.   Options include defining a threat model, limiting the blast radius of an attack by implementing separate security zones, the use of encryption at rest and in-flight and FIPS 140-2 validated ciphers, hardened builds and default configuration, as well as user access controls and key management. Data retention and secure deletion are also addressed. The very process of containerization creates additional security benefits with lightweight separation of domains.  Rook makes the process of applying hardening options easier, as this becomes a matter of simply modifying a .yaml file with the appropriate security context upon creation, making it a snap to apply the standard hardening options of Ceph to a container-based storage system.
    </td>
    <td>Federico Lucifredi
+  </td>
+  </tr>
+  <tr>
+   <td>14:35
```
Comment (jdurgin) [inline src/en/community/events/2022/ceph-virtual/index.md:286]: should this be a later time? 14:35 is the the previous talk

**Review by jdurgin — APPROVED** [review]
looks good once the time is fixed

**Pull request merged**